frontend/index.html

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <meta name="description" content="UPAI membership and tournament management portal">
```

Comment:
GG long time coming.. Can we add all meta tags for sharing and all

```
<!-- Primary Meta Tags -->
<title>The Hub | India Ultimate</title>
<meta name="title" content="The Hub | India Ultimate" />
<meta name="description" content="India Ultimate's membership and tournament management portal" />

<!-- Open Graph / Facebook -->
<meta property="og:type" content="website" />
<meta property="og:url" content="https://hub.indiaultimate.org/" />
<meta property="og:title" content="The Hub | India Ultimate" />
<meta property="og:description" content="India Ultimate's membership and tournament management portal" />
<meta property="og:image" content="https://metatags.io/images/meta-tags.png" />

<!-- Twitter -->
<meta property="twitter:card" content="summary_large_image" />
<meta property="twitter:url" content="https://hub.indiaultimate.org/" />
<meta property="twitter:title" content="The Hub | India Ultimate" />
<meta property="twitter:description" content="India Ultimate's membership and tournament management portal" />
<meta property="twitter:image" content="https://metatags.io/images/meta-tags.png" />

<!-- Meta Tags Generated with https://metatags.io -->
```

Reply:
![Hub Logo (1)](https://github.com/user-attachments/assets/0ba22f37-6089-47bf-8c87-f590f39171b9)

Will need to add it in assets and use this url in image tags

Reply:
Sending on zulip as well

Reply:
![Screenshot 2024-08-23 at 11 34 15 AM](https://github.com/user-attachments/assets/cbf69a16-7a2b-409b-8ec0-d2ade6f631cb)

Reply:
^ how it looks when we share

Reply:
ah cool, i'll add